docs/debug.md

```diff
@@ -85,4 +85,4 @@ server:
 ...
 ```
 
-The example configuration file is [here](../config/testdata/example_config.yaml). For more information, please refer to [config.go](./config/config.go).
+The example configuration file is [here](../config/test/data/example_config.yaml). For more information, please refer to [config.go](./config/config.go).
```

Comment:
same as above

Reply:
fixed in: 86fbb6a0eee66c10179c42c8cff2434e2f9715ee